src/http_servers/html-server.js

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs';
+import path from 'path';
+import http from 'http';
+
+const TEMPLATE_PATH = 'html/index.html';
+const customData = {
+  message: 'A custom message from html-server',
+};
+
+http.createServer((req, res) => {
+  res.writeHead(200, { 'Content-Type': 'text/html' });
+  const htmlTemplate = fs.readFileSync(path.join(__dirname, TEMPLATE_PATH), 'utf8');
+  res.end(generateHTML(htmlTemplate));
+})
+.listen(3002);
+
+http.createServer((req, res) => {
+  res.writeHead(200, { 'Content-Type': 'text/html' });
+  const chunks = [];
+  const htmlStream = fs.createReadStream(path.join(__dirname, TEMPLATE_PATH));
+  htmlStream.on('data', (chunk) => {
+    chunks.push(chunk);
+  });
+  htmlStream.on('end', () => {
```

Comment:
Good. That approach works good. 
Just in order to be aware how to do it with pipes:
`http.createServer((req, res) => {`
    `  res.writeHead(200, { 'Content-Type': 'text/html' });`
    `  fs.createReadStream(path.join(__dirname, TEMPLATE_PATH))`
      `    .pipe(replace('{message}', 'any new html content'))`
      `    .pipe(res);`
  `}).listen(3009);`
the only thing to add is 'stream-replace' npm package, that way we can leverage stream's pipeline opportunities

Reply:
Thanks for the approval @AliakseiBychyk. I have tried to use pipes but didn't get any clue on how to replace the string. Thanks for the suggestion I would make a note of it.

Reply:
You should install 'stream-replace' npm package and don't forget to add
`import replace from 'stream-replace'` to the code